docs/server/features/archiving_blog_post.md

```diff
@@ -121,22 +121,22 @@ Additionally, we recommend to use a more fine-grained AWS policy instead of givi
 Our configuration is almost complete!
 
 We just need to pass the AWS profile we created (`kurrentdb-archive`) as an environment variable to KurrentDB.
-If you are using the default AWS profile: `default`, you don't need to do these steps.
+If you are using the default AWS profile, `default,` you don't need to complete these steps.
 
 Specifying the `AWS_PROFILE` environment variable can be done as follows:
 ```
 $ sudo systemctl edit kurrentdb
 ```
 
-This will open up a text editor and you need to add these lines in the top section:
+This will open up a text editor, where you must add these lines in the top section:
 ```
 [Service]
 Environment="AWS_PROFILE=kurrentdb-archive"
 ```
 
 Save the file & exit.
 
-Finally, reload the systemd configuration by running:
+Finally, reload the system configuration by running:
```

Comment:
```suggestion
Finally, reload the _systemd_ configuration by running:
```

systemd wasn't a typo, it was referring to the systemd daemon: https://en.wikipedia.org/wiki/Systemd